modules/castbar.lua

```diff
@@ -85,6 +85,8 @@ local CastBar = GladiusEx:NewGladiusExModule("CastBar",
 
 function CastBar:OnEnable()
 	self:RegisterEvent("UNIT_SPELLCAST_START")
+	self:RegisterEvent("UNIT_SPELLCAST_EMPOWER_START")

```

Comment:
you can alias (by using a 2nd argument). See `UNIT_SPELLCAST_FAILED` below.
I think both EMPOWER events can just use the same code as normal `UNIT_SPELLCAST_*` events.